dotnetv3/README.md

```diff
@@ -40,110 +49,61 @@ In general, follow these steps:
-                    }
-                })
-                .Returns((CreateTableRequest r, CancellationToken token) =>
-                {
-                    return Task.FromResult(new CreateTableResponse { HttpStatusCode = HttpStatusCode.OK });
-                });
-
-            return mockDynamoDBClient.Object;
-        }
-
-        [Fact]
-        public async void CheckCreateTable()
-        {
-            IAmazonDynamoDB client = CreateMockDynamoDBClient();
-
-            var result = await CreateTable.MakeTableAsync(client, _tableName);
-
-            bool ok = result.HttpStatusCode == HttpStatusCode.OK;
-            Assert.True(ok, "Could NOT create table " + _tableName);
-        }
-    }
-}
+dotnet test -l "console;verbosity=detailed"
 ```
 
-To run this test,
-navigate to the **CreateTableTest** folder and run:
-
+To specify either unit or integration tests only, use the following category filters with the desired verbosity:
 ```
-dotnet test
+dotnet test --filter Category=Unit -l "console;verbosity=detailed"
 ```
-
-If you want more information, run:
-
+or
 ```
-dotnet test -l "console;verbosity=detailed"
+dotnet test --filter Category=Integration -l "console;verbosity=detailed"
 ```
 
 ## Docker image (Beta)
 This example code will soon be available in a container image
 hosted on [Amazon Elastic Container Registry (ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html). This image will be pre-loaded 
-with all .NET examples with dependencies pre-resolved, allowing you to explore
+with all .NET examples ready to build and run, allowing you to explore
 these examples in an isolated environment.
 
-⚠️ As of January 2023, the [SDK for .NET v3 image](https://gallery.ecr.aws/aws-docs-sdk-examples/dotnetv3) is available on ECR Public but is still
+⚠️ As of February 2023, the [SDK for .NET v3 image](https://gallery.ecr.aws/aws-docs-sdk-examples/dotnetv3) is available on ECR Public but is still
 undergoing active development. Refer to 
 [this GitHub issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/4126) 
 for more information. 
 
+### Build the Docker image
+
+1. Install and run docker on your machine.
+2. Navigate to the same directory as this readme.
+3. Optionally un-comment the .NET 5 setup in the docker file. .NET 6 will be installed by default.
+3. Run `docker build -t <image_name> .` where `image_name` is a name you provide for the image.
+
+### Launch the Docker container
+
+1. Run `docker run -it -v <your_credentials_folder_path>/.aws/credentials:/root/.aws/credentials <image_name>`. `-it` launches an
+   interactive terminal. `-v <your_cred...` is optional but recommended. It will mount your local credentials
+   file to the container.
+2. The terminal initiates a bash instance at the root of the container. Run `cd dotnetv3` and then you
+   can run examples and tests by navigating to a service folder and following the README instructions there. 
```

Comment:
Update to second sentence in step 2: 
 Run `cd dotnetv3`. Then, you can run examples and tests by navigating to a service folder and following the README instructions there.

Reply:
Fixed.